ts/store/reducers/profile.ts

```diff
@@ -75,6 +75,30 @@ export const profileNameSelector = createSelector(
     )
 );
 
+/**
+ * Return the surname of the profile if some, else undefined
+ */
+export const profileSurnameSelector = createSelector(
+  profileSelector,
+  (profile: ProfileState): string | undefined =>
+    pot.getOrElse(
+      pot.map(profile, p => capitalize(p.family_name)),
+      undefined
+    )
+);
+
+/**
+ * Return the birth date of the profile if some, else undefined
+ */
+export const profileBirthDateSelector = createSelector(
```

Comment:
A doubt only on these parts that are external to the it-wallet feature. In cherry pick logic, perhaps it would be better to keep them separate in order to bring them to master regardless of the feature.